src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="container">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
```

Comment:
main 
```suggestion
    <main class="container">
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></li>
        <div class="stars__star"></div>
      </div>
```
stars row shouldn't be a list. it is just a row of stars and not have any additional information for which will be nessasary to make a list.